clr-installer/main.go

```diff
@@ -349,21 +349,27 @@ func execute(options args.Args) error {
 		}
 	}
 
+	// The kbd bundle is an also-add and may be removed
 	if md.Keyboard != nil && !keyboard.IsValidKeyboard(md.Keyboard) {
-		return fmt.Errorf("Invalid Keyboard '%s'", md.Keyboard.Code)
+		msg := fmt.Sprintf("Warning: Host failed to validate keyboard: %s", md.Keyboard.Code)
+		fmt.Println(msg)
```

Comment:
What happens in the TUI/GUI when we print to the screen?

Reply:
TUI: The warning is written to stdout on the terminal.
GUI: Untested

There are other instances of fmt.Println and fmt.Printf in this function, so it should behave similarly.,

Reply:
Okay.